.github/workflows/ci-app_tests.yml

```diff
@@ -0,0 +1,139 @@
+name: CI App Tests
+
+# see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+on: # Trigger the workflow on push or pull request, but only for the master branch
+  push:
+    branches:
+      - "master"
+  pull_request:
+    paths:
+      - "src/lightning_app/**"
+      - "tests/tests_app/**"
+      - "requirements/app/**"
+      - "setup.py"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  pytest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macOS-10.15, windows-2019]
+        python-version: [3.8]
+        requires: ["oldest", "latest"]
+
+    # Timeout: https://stackoverflow.com/a/59076067/4521646
+    timeout-minutes: 20
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set min. dependencies
+        if: matrix.requires == 'oldest'
+        run: |
```

Comment:
lets use https://github.com/Lightning-AI/lightning/blob/f65aa7471921ab78f2ed131d3692507b1a06e3de/.actions/assistant.py#L66
check existing test full CI workflow

Reply:
can we do it in the following PR ?